src/main/java/teammates/storage/api/FeedbackResponseCommentsDb.java

```diff
@@ -306,7 +316,12 @@ public FeedbackResponseCommentAttributes updateFeedbackResponseComment(FeedbackR
             throw new EntityDoesNotExistException(ERROR_UPDATE_NON_EXISTENT + newAttributes.toString());
         }
         
-        frc.setGiverEmail(newAttributes.giverEmail);
+        if (isNewComment) {
+            frc.setGiverEmail(newAttributes.giverEmail);     
+        } else {
+            frc.setLastEditorEmail(newAttributes.giverEmail);
```

Comment:
is it possible to remove this branch for isNewComment. and thus remove add...comment and undo the rename of update...comment